docs/patterns.md

```diff
@@ -152,54 +167,69 @@ One of the awesome features of Hubot is its ability to make changes to a product
 
 There are a variety of different patterns for restricting access that you can follow depending on your specific needs:
 
-* Two buckets of access: full and restricted with whitelist/blacklist
+* Two buckets of access: full and restricted with include/exclude list
 * Specific access rules for every command (Role-based Access Control)
-* Blacklisting/whitelisting commands in specific rooms
+* Include/exclude listing commands in specific rooms
```

Comment:
Just wanted to call this out, thanks for making the change! It is a change I try to do whenever I notice it.